addons/Makefile

```diff
@@ -17,6 +17,11 @@ IMG_DEFAULT_NAME_TAG := $(IMG_DEFAULT_NAME):latest
 
 IMG_VERSION_OVERRIDE ?= $(shell git describe --always --dirty --tags)
 
+#Adding Support for GOPROXY and GOSUMDB
+GOPROXY ?= "https://proxy.golang.org,direct"
+GOSUMDB ?= off
+DISTROLESS_BASE_IMAGE ?= gcr.io/distroless/static:nonroot
+
```

Comment:
line no: 4 defines `common.mk`, if you move these changes from individual Makefile to `common.mk` you don't have to replicate these lines for each Makefile

Reply:
removed only from the files which have common.mk.
It can be removed from other make if it's not conflicting other vars